simapi/sim_api.rb

```diff
@@ -61,8 +62,7 @@ def request_is_not_from_simulator
 
 def update_latest(request)
   parsed_command_id = request.params["latest"].to_i || -1
-
-  File.write("./latest_processed_sim_action_id.txt", parsed_command_id.to_i, mode: "w") if parsed_command_id != -1
+  Latest.set(parsed_command_id)
```

Comment:
Sker det nogensinde at latest parameter ikke sendes med? Før denne ændring ville vi ikke opdatere latest filen hvis command id var -1 (Ikke sendt med), men nu bliver latest value vel bare overskrevet til at være -1?

Tænker at vi enten gør noget i denne stil
```suggestion
  Latest.set(parsed_command_id) if parsed_command_id != 1
```
Eller i stedet ikke defaulter parsed_command_id til at være -1, og lader den fejle i Latest.set siden den er nil

Tanker på dette er velkomne :)

Reply:
Går op for mig jeg lidt har misforstået din kommentar - du har helt ret i at det er dumt den bliver overskrevet. Men tror stadig casen er meget snæver

Reply:
Så tror jeg din suggestion er den bedste løsning

Reply:
Se nyeste commit